hello-app-iap/cloudbuild.yaml

```diff
@@ -0,0 +1,5 @@
+steps:
```

Comment:
no need for this file for 2 reasons:
1. I'll be pushing it
2. you can do the same thing without this file like `gcloud container builds submit -t TAG .`